include/deal.II/base/tensor_base.h

```diff
@@ -1,1552 +0,0 @@
-// ---------------------------------------------------------------------
```

Comment:
Nah, I don't think so. I think we need to keep the header file for at least one release.